concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2195,11 +2195,6 @@ jobs:
 
                 cf push paas-billing
 
-                # TODO: remove after the new route was deployed to staging/prod
-                if cf app paas-billing | grep -q "paas-billing.${APPS_DNS_ZONE_NAME}"; then
-                  cf delete-route -f "${APPS_DNS_ZONE_NAME}" --hostname paas-billing
-                fi
-
```

Comment:
I don't think `APPS_DNS_ZONE_NAME` is used by this task any more, so it should probably be removed from `params`

Reply:
Good catch. I've pushed an additional commit to remove it.